inputs_temp/XICS_allshots_C34.py

```diff
@@ -1,9 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've tried to find an error on a multi-line statement, like the failing result said but didn't find anything yet

Reply:
I've found it, can you try fixing the remaining non-confiormities (PEP8) in  tofu/spectro/_fit12d_funccostjac.py ?

Reply:
I've already done it last thursday but i will redo

Reply:
(it was a hidden syntax error: a parenthesis was not closed)